### PR TITLE
Subteams data on the team member page

### PR DESCRIPTION
**Pull request by jakob223**
TODO:
 - [x] expandable component for teams we are in
 - [x] merge with Piotr's RPC / data layer work

### CODE REVIEW

**Comment by jakob223** [comment]
cc @keybase/y2ksquad

File: shared/teams/team/member/index.new.tsx [hunk 1]
```diff
@@ -7,35 +7,200 @@ import * as Container from '../../../util/container'
...
+        <SubteamNotInRow teamID={teamID} subteam={item} idx={index} username={username} />
+      ),
+    },
+  ]
+  return (
+    <Kb.SectionList
+      stickySectionHeadersEnabled={true}
+      renderSectionHeader={({section}) => <Kb.SectionDivider label={section.title} />}
+      sections={sections}
+      keyExtractor={(item: Types.TeamMeta) => `member:${username}:${item.teamname}`}
```
Comment (buoyad) [inline shared/teams/team/member/index.new.tsx]: I'm seeing these in the console

> Warning: Encountered two children with the same key, `member:ayoubd:undefined:`

File: shared/teams/team/member/index.new.tsx [hunk 2]
```diff
@@ -7,35 +7,200 @@ import * as Container from '../../../util/container'
...
+            props: {teamID: props.subteam.id, username: props.username},
+            selected: 'teamsAddMemberToChannels',
+          },
+        ],
+      })
+    )
+  const onKickOut = () =>
+    dispatch(TeamsGen.createRemoveMember({teamID: props.subteam.id, username: props.username}))
+
+  const [expanded, setExpanded] = React.useState(false)
```
Comment (buoyad) [inline shared/teams/team/member/index.new.tsx]: :fire: No state in list items

File: shared/teams/team/member/index.new.tsx (hunk 1 above)
Comment (jakob223) [reply shared/teams/team/member/index.new.tsx]: I didn't realize the keyExtractor also applies to the section headers, that's surprising. Fixed.

**Comment by jakob223** [comment]
- [ ] after code freeze

File: shared/teams/team/member/index.new.tsx [hunk 3]
```diff
@@ -7,35 +7,226 @@ import * as Container from '../../../util/container'
...
+      <Kb.Button label="Add" onClick={() => setOpen(!open)} />
+    </FloatingRolePicker>
+  )
+
+  const memberCount = props.subteam.memberCount ?? -1
+  const body = (
+    <Kb.Box2 direction="vertical" alignItems="flex-start">
+      <Kb.Text type="BodySemibold">{props.subteam.teamname}</Kb.Text>
+      <Kb.Text type="BodySmall">
+        {memberCount.toLocaleString()} {pluralize('member', memberCount)}
```
[the remark this answers is not in the document]
Comment (jakob223) [reply shared/teams/team/member/index.new.tsx]: `memberCount` is 0 for empty teams, not -1. Might show `-1` for a subteam that hasn't had its data loaded, I guess.

**Pull request merged**